File: problem1/p1.c
/*
 * p1.c
 *
 * Zeke Reyna
 *
 * This function finds the sum all multiples of 3 and 5 below 1000.
 */

#include <stdio.h>
#include <stdlib.h>
#include <string.h>

#define BOUND 10

int main( int argc, char **argv )
{
	int sum;
	int i;
	int bound;

	(argc == 2) ? (bound = atoi( argv[1] )) : (bound = BOUND);

	sum = 0;
	for( i = 0; i < bound; i++ ) {
		if( i % 3 == 0 || i % 5 == 0 )
			sum += i;
	}

	printf( "\nthe sum of all multiples of 3 and 5 below 1000" );
	printf( " is %d\n\n" , sum );

	return 0;
}

File: problem1/vers_py/p1.py
### p1.py
#
# Zeke Reyna
# 
# This function sums all the multiples of 3 and 5
# below BOUND

import sys

def main(argv):
    if len(argv) == 2:
        bound = int(argv[1])
    else:
        bound = 10

    sum = 0
    for i in range(1, bound):
        if i % 3 == 0 or i % 5 == 0:
            sum += i

    print("\nThe sum of all multiples of 3 and 5 below ", bound, " is ", sum , "\n" )



main(sys.argv)
